scripts/setSwapFee.js

```diff
@@ -0,0 +1,76 @@
+// ES5 style
+const config = require("./config/deploymentConfig");
+const thorify = require("thorify").thorify;
+const Web3 = require("web3");
+const Factory = require(config.pathToV2FactoryJson);
+const Pair = require(config.pathToV2PairJson);
+const assert = require('assert');
+const readlineSync = require("readline-sync");
+
+let network = null;
+let pairAddress = null;
+let swapFee = null;
+
+if (process.argv.length !== 5) 
+{
+    console.error("Usage: node setSwapFee.js [mainnet|testnet] [pair address] [swapFee in basis points (100 means 1%)]");
+    process.exit(1);
+} 
+else
+{
+    network = config.network[process.argv[2]];
+    if (network === undefined) {
+        console.error("Invalid network specified");
+        process.exit(1);
+    }
+
+    if(!Web3.utils.isAddress(process.argv[3]))
+    {
+        console.error("Invalid address for pair contract");
+        process.exit(1);
+    }
+
+    pairAddress = process.argv[3];
+    swapFee = process.argv[4];
+}
+
+const web3 = thorify(new Web3(), network.rpcUrl);
+web3.eth.accounts.wallet.add(config.ownerPrivateKey);
```

Comment:
How is this set? Shouldn't the config load from `.env`?

Reply:
yes the config file (`deploymentConfig.js`) loads the private key from `.env`